### PR TITLE
Add date column to attachments

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/attachments/templates/attachments/attachments.html [hunk 1]
```diff
@@ -29,6 +30,7 @@
                   {% endif %}
                 </td>
                 <td>{% field form.description is_admin=False show_label=False %}</td>
+                <td>{% if obj.created %}{{ obj.created }}{% else %}&mdash;{% endif %}</td>
```
Comment (mkurek) [inline src/ralph/attachments/templates/attachments/attachments.html]: is there any case when `obj.created` will be evaluated as `False`?

Comment (szok) [inline src/ralph/attachments/templates/attachments/attachments.html]: yes because its work as inline and obj is empty

**Comment by mkurek** [comment]
:ok:

**Comment by vi4m** [comment]
LGTM

**Pull request merged**